deflacue/deflacue.py

```diff
@@ -249,8 +249,17 @@ def process_cue(self, *, cue_file: Path, target_path: Path):
         cd_info = cue.meta.data
         tracks = cue.tracks
 
-        def sanitize(val: str) -> str:
-            return val.replace('/', '')
+        def _sanitize(val: str, illegals: tuple) -> str:
+            for i in illegals:
+                val = val.replace(i, '')
+            return val
+
+        if os.name == 'nt':
```

Comment:
Let's define an OS appropriate sanitize function at class level (so it's constructed once at module import level).
Also this function will require some tests with os.name mocking to ensure the behaviour.

Reply:
Will get around to it this weekend. FYI, the existing tests seem to fail on my Windows machine (has something to do with how the encoded Russian language .cues are loaded, didn't look too much further into it). Will try to investigate it.